godot-core/src/registry/signal/connect_handle.rs

```diff
@@ -54,7 +54,9 @@ impl ConnectHandle {
     /// [`disconnect()`][Self::disconnect] -- e.g. through [`Signal::disconnect()`][crate::builtin::Signal::disconnect] or
     /// [`Object::disconnect()`][crate::classes::Object::disconnect].
     pub fn is_connected(&self) -> bool {
-        self.receiver_object
-            .is_connected(&*self.signal_name, &self.callable)
+        self.receiver_object.is_instance_valid()
```

Comment:
I would suggest something along the lines of: 

```suggestion
        if !self.receiver_object.is_instance_valid() {
                 godot_warn!("...");
                 return false
                 }       
```

1) This is something that probably comes from logic error which should be properly addressed in the userspace and hiding it from user kinda sucks :sweat_smile: 
2) We can always easily slap `#[cfg(debug_assertions)]` on it later.